docs/update_tails_usbs.rst

```diff
@@ -0,0 +1,104 @@
+  #. Install the :ref:`latest version of Tails <Step 2>` on your *primary Tails USB* or a blank USB stick.
+  #. Use the *primary Tails USB* to :ref:`perform a manual update of your desired Tails USB <Step 3>` on a fully offline (airgapped) computer, such as the computer used for the *Secure Viewing Station*.
+
+What you need
+^^^^^^^^^^^^^
+
+  #. Your *Admin Workstation* computer
+  #. A *primary Tails USB* stick (you may still have one; it was used to create the *Admin Workstation*, *Secure Viewing Station*, and *Journalist Workstation* Tails USBs during the initial SecureDrop install process)
+  #. The Tails USB that you want to update
+  #. A Backup USB to back up the data on your existing Tails USBs
+  #. An airgapped computer, such as the computer used for the *Secure Viewing Station*.
+
+.. _Step 1:
+
+1. Back up your Tails USB
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`Follow the instructions`_ to back up your existing Tails USB.
+
+.. _Follow the instructions: https://docs.securedrop.org/en/stable/backup_workstations.html
+
+.. _Step 2:
+
+2. Get the latest version of Tails
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you have an existing *primary Tails USB*, boot into it on your *Admin Workstation* computer and follow the graphical updater prompts that guide you through the `update process`_.
+
+Alternatively, you can also download and `install the newest version of Tails from scratch`_ (as you did when you first installed SecureDrop). This may be faster if your *primary Tails USB* has not been updated in a while.
+
+.. _update process: https://tails.boum.org/doc/first_steps/upgrade/index.en.html
+
+.. _install the newest version of Tails from scratch: https://docs.securedrop.org/en/stable/set_up_tails.html#install-tails
+
+.. _Step 3:
+
+3. Perform airgapped update
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this step, use the up-to-date *primary Tails USB* to update your desired Tails USB.
+
+.. warning::
+        The entire *Secure Viewing Station* is designed to be airgapped, so
+        the *SVS* Tails USB should **never** be plugged into a computer with
+        a network connection.
+
+        Use an airgapped computer, such as the *Secure Viewing Station* computer, to perform the steps in this section.
+
+Plug your *primary Tails USB* into the airgapped computer and boot into Tails.
+
+You can then perform the `manual update steps`_.
+
+.. _manual update steps: https://tails.boum.org/upgrade/clone-overview/index.en.html
+
+
+If you encounter issues
+^^^^^^^^^^^^^^^^^^^^^^^
+
+If you run into issues, you can always restore your data from the backup device following the instructions `here`_.
+
+If you continue to have problems, you can contact us through the `SecureDrop Support Portal`_.
+
+.. _SecureDrop Support Portal: https://securedrop-support.readthedocs.io/en/latest/
+
+.. _here: https://docs.securedrop.org/en/release-0.14.0/backup_workstations.html#restoring-a-workstation-from-a-backup
```

Comment:
Link should be internal rather than to a specific published version of the docs.

Reply:
:white_check_mark: